core/src/main/java/org/apache/accumulo/core/util/LocalityGroupUtil.java

```diff
@@ -50,16 +52,16 @@
 
   // private static final Logger log = Logger.getLogger(ColumnFamilySet.class);
 
-  public static final Set<ByteSequence> EMPTY_CF_SET = Collections.emptySet();
+  public static final Set<ByteSequence> EMPTY_CF_SET = ImmutableSet.of();
```

Comment:
What ImmutableSet comparison logic? re-skimming your patch, it's not jumping out at me.